cmd/preflight/cli/run.go

```diff
@@ -135,10 +135,11 @@ func runPreflights(v *viper.Viper, arg string) error {
 		go func() {
 			for {
 				select {
-				case _, ok := <-progressCh:
+				case msg, ok := <-progressCh:
 					if !ok {
 						return
 					}
+					fmt.Fprintln(os.Stderr, msg)
```

Comment:
progressCh should probably be a string type.  Since it's an interface this could panic if you sent it a nil. Maybe use fmt.Fprintf(os.Stderr, "%v" , msg) instead

Reply:
updated